tedana/decomposition/eigendecomp.py

```diff
@@ -32,7 +32,7 @@
     """),
     description='Introduces method for choosing PCA dimensionality '
                 'automatically')
-def run_svd(data):
+def run_svd(data, mle=True):
```

Comment:
We'll need to split this somehow so that the MLE paper is only cited when MLE is used, but I'm not sure what the best way to do that is.

Reply:
Opened an issue on duecredit ! https://github.com/duecredit/duecredit/issues/141

Reply:
In the interest of getting this PR ready to merge so we can work on validation analyses, I was thinking of just running the regular PCA in `tedpca` and calling the MLE-specific function only if that's requested. We could circle back to the organization later. WDYT?